tensorboard/plugins/hparams/tf_hparams_parallel_coords_plot/tf-hparams-parallel-coords-plot.ts

```diff
@@ -269,7 +269,8 @@ class TfHparamsParallelCoordsPlot extends LegacyElementMixin(
         if (!this.options.configuration.columnsVisibility[colIndex]) {
           continue;
         }
-        if (utils.columnValueByIndex(schema, sg, colIndex) === undefined) {
+        const columnValue = utils.columnValueByIndex(schema, sg, colIndex);
```

Comment:
Filters out session groups with NaNs so that they are not rendered in the parallel coordinates view.